packages/expo-screen-orientation/android/build.gradle

```
@@ -70,5 +70,5 @@ if (new File(rootProject.projectDir.parentFile, 'package.json').exists()) {
 dependencies {
   unimodule 'unimodules-core'
 
-  api "org.jetbrains.kotlin:kotlin-stdlib-jdk7:1.3.41"
+  api "org.jetbrains.kotlin:kotlin-stdlib-jdk7:1.4.21"
```

Would be nice to keep these versions in-sync 😉 
```suggestion
  api "org.jetbrains.kotlin:kotlin-stdlib-jdk7:${safeExtGet("kotlinVersion", "1.4.21")}"
```